packages/ove-service-auth/README.md

```diff
@@ -0,0 +1,191 @@
+# OVE Base Proxy
+
+This is a generic base proxy that supports various configurations, provided either via environment variables or 
+configuration files.
+
+## Configurations
+
+### Request Proxy
+
+The request proxy supports two configurations: a global or service-based proxy. The service proxy supports basic
+load balancing.
+
+The proxy mechanism can be enabled by setting the **SERVICE_PROXY_ENABLED** environment variable, the default is **True**.
```

Comment:
Have you considered using a `PROXY_TYPE` variable that can be `global` or `service` instead?

Reply:
Even if you add a proxy type variable, you still need to provide the global proxy url